app/views/layouts/_flash.html.erb

```diff
@@ -2,6 +2,6 @@
   <div class="alert alert-<%= name %> alert-dismissable" role="alert">
     <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
     <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-    <span class="sr-only">Message: </span><%= msg %>
+    <span class="sr-only">Message: </span><%= sanitize(msg, tags: %w(a), attributes: %w(href rel data-method)) %>
```

Comment:
@njgheorghita Sanitize was the key. That is a built-in helper method that allows us to configure what tags and attributes are passed through.